README.md

```diff
@@ -57,6 +57,40 @@ store = Win32::Certstore.open("Root")
 store.add(certificate_file_path)
 ```
 
+### Delete certificate
+
+Deletes a valid certificate from valid certificate store and returns output in string format.
+
+**Notes: The certificate_name should be valid `Issuer CN name` of the certificate**
```

Comment:
I think you mean that `certificate_name` should be the CN or Common Name of the certificate? You could have many certificates with the same issuer.